src/screens/AppIntroScreen.tsx

```diff
@@ -158,69 +316,39 @@ export const AppIntroScreen = ({ setOnboardingComplete, onlyTermsAndConditions }
   }
 
   return (
-    <SafeAreaViewFlex style={styles.background}>
+    <SafeAreaViewFlex style={[styles.background, { backgroundColor }]}>
       <StatusBar style="dark" translucent backgroundColor="transparent" />
       <AppIntroSlider<AppIntroSlide>
         activeDotStyle={onlyTermsAndConditions ? styles.hiddenDot : styles.activeDot}
+        bottomButton
         data={slides}
         dotClickEnabled={false}
         dotStyle={onlyTermsAndConditions ? styles.hiddenDot : styles.inactiveDot}
         keyExtractor={keyExtractor}
         onDone={setOnboardingComplete}
-        onSlideChange={(_, index) => {
-          // hide the next button on the terms and conditions slide
-          if (
-            index + 1 ==
-            slides.findIndex((slide) => slide.onLeaveSlideName === Initializer.TermsAndConditions)
-          ) {
-            setShowNextButtonTermsAndConditions(false);
-          }
-
-          slides[index]?.onLeaveSlide?.(true);
-        }}
```

Comment:
this is entirely gone? what about the `showNextButtonTermsAndConditions` logic? did you try the case, where it was needed before with the new code?

Reply:
I don't think we need this logic anymore, because we already create the buttons with the props. This logic was used to show buttons outside the `AppIntroSlider` component.